README.md

```
@@ -1,8 +1,13 @@
-# spotatui
-
-> A terminal music player written in Rust, powered by [Ratatui](https://github.com/ratatui-org/ratatui) — native Spotify streaming, synced lyrics, a real-time audio visualizer, and optional Local, Subsonic/Navidrome, Internet Radio, and YouTube sources. Spotify is optional.
->
-> A community-maintained, actively developed fork of [spotify-tui](https://github.com/Rigellute/spotify-tui).
+<h1>
+    <p align="center">
+        <img src="https://cdn.discordapp.com/icons/1534820237122207815/e54d80b8280857dab9b4be463ec884a7.webp?size=80&quality=lossless"><br>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)